### DISCUSSION TITLE
에디터 인스펙터 창에 텍스처 할당 불가 문제

### DISCUSSION THREAD

**Discussion by Asipe42** [naverz/zepeto-studio-kor#1181] [5. Q&A]
ArgumentNullException: Value cannot be null.
Parameter name: source 오류가 나면서

![image](https://user-images.githubusercontent.com/86538738/185017135-602c0eed-07cc-4b0c-855b-05acd8c0cb25.png)
인스펙터 창에서 텍스처를 할당하지 못하는 문제가 발생합니다!

**Comment by Asipe42** [comment]
텍스처 뿐만 아니라 필드 속성 관계 없이 무작위로 할당 불가능한 문제가 발생합니다!

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/issues/979 와 동일한 이슈로 내부에서 확인 중에 있습니다. 
감사합니다.
